angular.json

```diff
@@ -58,7 +58,7 @@
                 {
                   "type": "anyComponentStyle",
                   "maximumWarning": "2kb",
-                  "maximumError": "4kb"
+                  "maximumError": "4.2kb"
```

Comment:
mysle ze obejdziemy sie bez zmiany tego parametru

Reply:
Nie mam pomysłu jak zmniejszyć rozmiar tego pliku, on jest duży bo każda wersja karty wymaga troche innego stylowania, ze względu na inny background. Próbowałem wyodrębniać kod do innych klas aby style się nie powtarzały, ale to ciągle za mało.

Reply:
udało się !!! 🥳